### ISSUE TITLE
PSR12.Operators.OperatorSpacing false positive

### ISSUE THREAD

**Issue by morozov** [squizlabs/PHP_CodeSniffer#3098]
**Describe the bug**
The sniff doesn't recognize unary minus applied to a number used with the`exit` language construct.

**Code sample**
```php
<?php

function test()
{
    exit -1;
}
```

**Custom ruleset**
None.

**To reproduce**
1. Run `phpcs -s --standard=PSR12 --sniffs=PSR12.Operators.OperatorSpacing test.php`
```
FILE: /home/morozov/Projects/phpcs-playground/test.php
------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 5 | ERROR | [x] Expected at least 1 space after "-"; 0 found
   |       |     (PSR12.Operators.OperatorSpacing.NoSpaceAfter)
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------

Time: 33ms; Memory: 6MB
```
2. Run `phpcbf --standard=PSR12 --sniffs=PSR12.Operators.OperatorSpacing test.php`
```
PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
/home/morozov/Projects/phpcs-playground/test.php      1      0
----------------------------------------------------------------------
A TOTAL OF 1 ERROR WERE FIXED IN 1 FILE
----------------------------------------------------------------------

Time: 37ms; Memory: 6MB
```

The resulting file contents are:
```php
<?php

function test()
{
    exit - 1;
}
```

**Expected behavior**
No error is reported.

**Versions (please complete the following information):**
 - OS: Linux
 - PHP: 7.4.9
 - PHPCS: 3.5.6
 - Standard: PSR-12
